lms/templates/verify_student/photo_reverification.html

```diff
@@ -62,7 +62,7 @@ <h3 class="title">${_("Error submitting your images")}</h3>
         <div class="message">
           <h3 class="title">${_("Please Resubmit Your Verification Information")}</h3>
           <div class="copy">
-            <p>${_("There was an error with your previous verification. In order proceed in the verified certificate of achievement track of your current courses, please complete the following steps.")}</p>
+            <p>${_("In order proceed in the verified certificate of achievement track of your current courses, please complete the following steps.")}</p>
```

Comment:
Like, whoa - nope, I definitely haven't seen that text before. Hold on a minute and I'll have a recommendation.